openkfw/TruBudget
# 991 Default error handling - excel-export

### Checklist




### Description

Add default error handling in excel export

### How to test

1. Start full environment
2. Simulate error in download route
3. Check if the http response is correct with necessary information
4. Check if the logs make sense and if they are descriptive.
